src/Ed25519VerificationKey2018.js

```diff
@@ -339,8 +353,11 @@ function _isEqualBuffer(buf1, buf2) {
   }
   return true;
 }
-
+// Used by CryptoLD harness for dispatching.
 Ed25519VerificationKey2018.suite = SUITE_ID;
+// Used by CryptoLD harness's fromKeyId() method.
+Ed25519VerificationKey2018.SUITE_CONTEXT =
+  'https://w3id.org/security/suites/ed25519-2018/v1';
```

Comment:
Shouldn't `fromKeyId` just be implemented within this very class?

Reply:
So, I thought about that approach initially. But then I realized that it would be the exact same method, copied and pasted to every suite. They all do the same things:
* Validate the input params
* Call documentLoader
* Check the resulting key doc for context, type and revoked
* Call the appropriate suite's `from()` method.

And all the key-specific stuff is in the `from()` method anyway. So, why not just implement `fromKeyId()` once, at the parent CryptoLD level?